credentials/vcdatamodel/validators.go

```diff
@@ -0,0 +1,38 @@
+package vcdatamodel
+
+import (
+	"errors"
+	"fmt"
+)
+
+func validateContext(c []URI) error {
+	err := errors.New(fmt.Sprintf("@context is missing default context %s", DefaultContext))
+	for _, v := range c {
+		if v == DefaultContext {
+			return nil
+		}
+	}
+
+	return err
+}
+
+func validateVCType(t []string) error {
+	err := errors.New(fmt.Sprintf("type is missing default %s", DefaultVCType))
+	for _, v := range t {
+		if v == DefaultVCType {
+			return nil
+		}
+	}
+
+	return err
+}
+
+func validateCredentialSubject(cs []CredentialSubject) error {
```

Comment:
May need clarity on what to validate here, or perhaps some validation should move to the createVCOptions because the other web5 projects seem to imply that ID should be there on Credential Subject, but the data model clearly states that it is optional.

If it is not present (an empty string, for example), what should we be putting into the subject of the JWT claim when signing? Maybe for our creation use case it is not optional, and if not I should be validating that as well.